backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -796,37 +761,12 @@ void convertToDiagnosticReport_TestEvent_removedTestEvent() {
   }
 
   @Test
-  void convertToDiagnosticReport_TestEvent_matchesJson() throws IOException {
-    var internalId = "3c9c7370-e2e3-49ad-bb7a-f6005f41cf29";
-    var testEvent =
-        new TestEvent(
-            new TestOrder(
-                new Person(null, null, null, null, null),
-                new Facility(
-                    null,
-                    null,
-                    null,
-                    null,
-                    null,
-                    null,
-                    null,
-                    new DeviceSpecimenType(
-                        new DeviceType(null, null, null, "95422-2", null, 0), null),
-                    Collections.emptyList())),
-            false,
-            Collections.emptySet());
-    ReflectionTestUtils.setField(testEvent, "internalId", UUID.fromString(internalId));
-
+  void testEventNullDeviceType_convertToDiagnosticReport() {
+    var testEvent = TestDataBuilder.createEmptyTestEvent();
     var actual = convertToDiagnosticReport(testEvent);
 
-    String actualSerialized = parser.encodeResourceToString(actual);
-    var expectedSerialized =
-        IOUtils.toString(
-            Objects.requireNonNull(
-                getClass().getClassLoader().getResourceAsStream("fhir/diagnosticReport.json")),
-            StandardCharsets.UTF_8);
-
-    JSONAssert.assertEquals(actualSerialized, expectedSerialized, true);
+    assertThat(actual.getStatus()).isEqualTo(DiagnosticReportStatus.FINAL);
+    assertThat(actual.getCode().getCoding()).isEmpty();
```

Comment:
This json matching test never ended up getting replaced. One of the reasons I end up used the JSON matching is because there are a lot of fields on each FHIR model and it wouldn't be practical to ensure each other one was null.

Reply:
Ooh great catch! I added this test back and did another check against `main` to ensure all the correct tests are in place.